src/components/EditorAndOutput.vue

```diff
@@ -69,11 +73,13 @@ watchEffect(() => {
 const openFile = useOpenFile(
   // Open the first .wgsl file if it exists
   props.fs.listFiles().find((v) => v.endsWith(".wgsl")) ?? null,
-  props.fs
+  props.fs,
```

Comment:
Could you double-check if your IDE is configured correctly? The auto-formatter shouldn't add this comma